source/view.c

```diff
@@ -972,11 +1055,18 @@ static void rofi_view_refilter ( RofiViewState *state )
         tokenize_free ( state->tokens );
         state->tokens = NULL;
     }
+    if ( state->pattern ) {
```

Comment:
`state->pattern` is the complete pattern (`state->tokens` is its tokenized form) which is only used in `-matching fuzzy` mode.